.vscode/settings.json

```diff
@@ -8,5 +8,6 @@
     "python.testing.pytestEnabled": true,
     "python.formatting.provider": "black",
     "python.languageServer": "Microsoft",
-    "restructuredtext.confPath": "${workspaceFolder}/docs"
+    "restructuredtext.confPath": "${workspaceFolder}/docs",
+    "editor.defaultFormatter": null
```

Comment:
What issues do you have if you don't set editor.defaultFormatter to null?

Reply:
VS Code via SSH fails to format files with Black unless I have this setting. I can't find relevant logging or really understand why.